src/pages/core/specification/instantiate2-algo.mdx

```diff
@@ -0,0 +1,42 @@
+---
+tags: ["core", "specification"]
+---
+
+# Instantiate2 algorithm
+
+With the instantiate2 algorithm you can create a contract address in a
+predictable and deterministic way. The underlying algorithm is rather simple.
+
+We use SHA-256 as the underlying hashing algorithm.
+
+You need to provide the following inputs:
+
+- Checksum: This _has_ to be a SHA-256 hash
```

Comment:
what is it a hash of? in the example it's the string "checksum"

Reply:
It is supposed to be the hash of the contract module (so the hash of the Wasm bytes). Clarified it.